itest/lnd_coop_close_with_htlcs_test.go

```diff
@@ -65,6 +65,7 @@ func testCoopCloseWithHtlcs(ht *lntest.HarnessTest) {
 	closeClient := alice.RPC.CloseChannel(&lnrpc.CloseChannelRequest{
 		ChannelPoint: chanPoint,
 		NoWait:       true,
+		TargetConf:   6,
```

Comment:
So then does this already have implications to the existing/default RPC behavior?

Reply:
This will affect users who specify neither feerate nor conf target, which is unlikely? So the previous behavior is, when nothing specified, default to conf target of 6, while the current behavior is, you must specify either conf target or feerate.